.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file isn't needed.

Reply:
@paramsingh, I did not create it. I don't really know how it got created and can't delete as I don't know its location where it got created.